lib/mjsonwp/mjsonwp.js

```diff
@@ -245,6 +245,11 @@ function buildHandler (app, method, path, spec, driver, isSessCmd) {
         driverRes = null;
       }
 
+      // if the status is not 0,  throw the appropriate error for status code.
+      if (util.hasValue(driverRes) && util.hasValue(driverRes.status) && parseInt(driverRes.status, 10) !== 0) {
+        throw errorFromCode(driverRes.status, driverRes.value);
```

Comment:
I'm really confused why we'd want to throw something here rather than simply pass the (error) status back on to the client? Is it the case that we get MJSONWP objects from `driver.execute`? I thought we got actual responses